topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>    > Definition: Dereplicate reads into unique sequences
+>    > * compare all the reads in a data set to each other
+>    > * group strictly identical reads together
+>    > * output the sequence for each group and its count in the original dataset (in this way, all duplicated reads are removed)
+>    > *Definition adapted from Seguritan and Rohwer (2001)*
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many sequences you had and how many you finally obtain?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. From 43 326 to 3 962
+> >
+> {: .solution}
+>
+{: .question}
+
+
+## Limit number of informations with **obiannotate**
+
+> ### {% icon hands_on %} Hands-on: Adds/Edits sequence record annotations
+>
+> 1. {% tool [obiannotate](toolshed.g2.bx.psu.edu/repos/iuc/obi_annotate/obi_annotate/1.2.13) %} with the following parameters:
+>    - In *"Keep only attribute with key"*:
+>        - *"key"*: `count`
+>        - *"if you want to specify a second key"*: `merged_sample`
+
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > obiannotate is the command that allows adding/modifying/removing annotation attributes attached to sequence records.
+>    > Once such attributes are added, they can be used by the other OBITools commands for filtering purposes or for statistics computing.
+>    > 
+>    > Here, the goal is to keep only `count` and `merged_sample` key=value attributes! 
+>    {: .comment}
+>
+{: .hands_on}
+
+
+## Computes basic statistics for attribute values with **obistat**
+
+> ### {% icon hands_on %} Hands-on: Computes basic statistics for attribute values
+>
+> 1. {% tool [obistat](toolshed.g2.bx.psu.edu/repos/iuc/obi_stat/obi_stat/1.2.13) %} with the following parameters:
+>    - In *"Category attribute"*:
+>        - {% icon param-repeat %} *"Insert Category attribute"*
+>            - *"How would you specify the category attribute key?"*: `simply by a key of an attribute`
+>                - *"Attribute used to categorize the sequence records"*: `count`
+>    - *"Use a specific option"*: `no`
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > stats computes basic statistics for attribute values of sequence records. The sequence records can be categorized or not using one or several -c options. By default, only the number of sequence records and the total count are computed for each category. Additional statistics can be computed for attribute values in each category, like:
+>    > * minimum value (-m option)
+>    > * maximum value (-M option)
+>    > * mean value (-a option)
+>    > * variance (-v option)
+>    > * standard deviation (-s option)
+>    > 
+>    > The result is a contingency table with the different categories in rows, and the computed statistics in columns.
+>    {: .comment}
```

Comment:
Also this one. I think that the essential information should be included at the beginning of each section, and restrict the content of the comment blocks to accessory information; otherwise, it is a bit confusing.

Reply:
you're completely right! Thank you! Done!